backend/media/src/main/java/co/airy/core/media/services/MediaUpload.java

```diff
@@ -59,4 +74,29 @@ private String resolveContentType(final String fileName, final InputStream is) t
     private String appendSlash(String path) {
         return !path.endsWith("/") ? path + "/" : path;
     }
+
+    private void checkConnectionStatus() {
+        connectionStatus = false;
+        try {
+            List<Bucket> buckets = amazonS3Client.listBuckets();
+            if (buckets != null) {
+                log.info("check connection to S3 successful");
+                connectionStatus = true;
+            }
+        } catch (SdkClientException e) {
+            log.error("AWS SDK extension", e);
+
+        } catch (Exception e) {
+            log.error("unexpected extension", e);
```

Comment:
`extension` is that a typo?

Reply:
I wanted to write `exception` 🙈. I am bit dyslexic it won't be the last time this will happen 😅